### ISSUE TITLE
Inconsistent parsing of empty lists/maps

### ISSUE THREAD

**Issue by sinfex** [orientechnologies/orientdb#3519]
Am I missing something?

The following works:

```
CREATE VERTEX V SET empty = []
CREATE VERTEX V SET empty = {}
```

The following fails (mind **the space between curly/square brackets**):

```
CREATE VERTEX V SET empty = [ ]
CREATE VERTEX V SET empty = { }
```

**Comment by luigidellaquila** [comment]
Hi @sinfex 
this is a known issue and will be addressed in 2.1

**Comment by luigidellaquila** [comment]
fixed in 2.1 snapshot (develop branch)
